applications/NXmx.nxdl.xml

```diff
@@ -816,6 +822,27 @@
 						</doc>
 					</field>
 
+					<field name="flux_integrated" type="NX_FLOAT" units="NX_FLUX" minOccurs="0">
+						<doc>
+							Flux density incident on beam plane area in photons
+							per per unit area. Useful where temporal profiles of flux
```

Comment:
Typo: extra per

Reply:
Merge https://github.com/nexusformat/definitions/pull/1082 to fix.